webapp/src/app/c4dt/partner/partner.component.ts

```diff
@@ -22,6 +24,15 @@ export class PartnerComponent extends ContactsComponent {
     }
```

Comment:
While I'm at it, I guess we can remove this block, as the super will already do that.